packages/domain/lib/usecases/get_movie_cast_usecase.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:domain/entities/movie_character_entity.dart';
+import 'package:domain/repositories/images_repository.dart';
+import 'package:domain/repositories/movies_repository.dart';
+import 'package:domain/usecases/usecase.dart';
+
+class GetMovieCastUsecaseParams {
+  final int movieId;
+  final int maxNumberOfActors;
+
+  GetMovieCastUsecaseParams(
+    this.movieId,
+    this.maxNumberOfActors,
+  );
+}
+
+class GetMovieCastUsecase extends UseCaseParams<GetMovieCastUsecaseParams,
+    Future<Iterable<MovieCharacterEntity>>> {
```

Comment:
why iterable? seems that u can use List directly, no?

Reply:
Fixed.